telco-ran/configuration/machineconfigs/misc/75-rhcos4-e8-master-audit-rules-dac-modification-chmod.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+spec:
+  config:
+    ignition:
+      version: 3.1.0
+    storage:
+      files:
+        - contents:
+            source: data:,-a%20always%2Cexit%20-F%20arch%3Db32%20-S%20chmod%20-F%20auid%3E%3D1000%20-F%20auid%21%3Dunset%20-F%20key%3Dperm_mod%0A-a%20always%2Cexit%20-F%20arch%3Db64%20-S%20chmod%20-F%20auid%3E%3D1000%20-F%20auid%21%3Dunset%20-F%20key%3Dperm_mod%0A
```

Comment:
can we please keep the human readable format for MCs?
see other MCs in this repo for examples on how to generate them..

Reply:
Other entries for this sort of `source` entry are using base64 encoding such as:
https://github.com/openshift-kni/telco-reference/blob/main/telco-ran/configuration/source-crs/extra-manifest/01-disk-encryption-pcr-rebind-worker.yaml#L19-L20

Should I be using that here as well?

Reply:
I created a new commit using a [script](https://github.com/sebrandon1/compliance-scripts/pull/6) that looks through the objects and writes a plaintext comment.

What do you think @yuvalk?  This would prevent us having to create a builder, base64 anything, etc.